modules/nw-udn-limitations.adoc

```diff
@@ -9,7 +9,10 @@
 While user-defined networks (UDN) offer highly customizable network configuration options, there are limitations that cluster administrators and developers should be aware of when implementing and managing these networks. Consider the following limitations before implementing a user-defined network.
 
 //Check on the removal of the DNS limitation for 4.18 or 4.17.z.
-* *DNS limitations*: DNS lookups for pods resolve to the pod's IP address on the cluster default network. Even if a pod is part of a user-defined network, DNS lookups will not resolve to the pod's IP address on that user-defined network.
+* *DNS limitations*:
+** DNS for UDN pods is unsupported. The Kubernetes API (KAPI) and external DNS are supported. 
+** DNS lookups for pods resolve to the pod's IP address on the cluster default network. Even if a pod is part of a user-defined network, DNS lookups will not resolve to the pod's IP address on that user-defined network.
+** When a pod is assigned to a primary user-defined network, routes are set up to allow the pod to reach the Kubernetes API (KAPI) and DNS services on the cluster's default network. All other traffic is blocked by firewall rules on this network. Although KAPI and DNS are accessible by default, users might optionally configure access to other Kubernetes services on the cluster default network as well, however, the pod's IP shown by KAPI will continue to reflect the IP assigned in the cluster default network. 
```

Comment:
>  When a pod is assigned to a primary user-defined network, routes are set up to allow the pod to reach the Kubernetes API (KAPI) and DNS services on the cluster's default network. 

I would prefer if we avoid mentioning routes here and just say that KAPI and the DNS service are accessible.

> users might optionally configure access to other Kubernetes services on the cluster default network as well

We do not currently have support for that.